### DISCUSSION TITLE
Volume Down/Up Key as CTRL (Optional)

### DISCUSSION THREAD

**Discussion by PatAbah** [Acode-Foundation/Acode#625] [Ideas]
I see all these cool shortcuts on the shortcut menu. However, sadly, I do not have a wireless keyboard to enjoy them on my mobile. I think it would be great to create a feature (or perhaps an extension) where volume keys can be used as the CTRL key.

That way Volume-xxx acts as CTRL-xxx
Or Volume-SHFT-xxx acts as CTRL-SHFT-xxx
![IMG_20230120_064142](https://user-images.githubusercontent.com/34959120/213625708-08f36e03-263f-4dcd-96c5-b2fa06e5cb65.jpg)

P.S:
The idea suggests that this feature can be disabled if user prefers to use Volume keys for scrolling instead/doesn't want any action binded to volume keys at all.

**Comment by PatAbah** [comment]
OK, Legend Sabbir just saved the day. You can search for "Command Button" in plugins and install to use the future I requested. It's not based on the volume up/down button I suggested. It's kinda better.

**Reply by bettywolfy** [reply]
@PatAbah You can now close, now in this new version the "command palette" has been changed to "ctrl" 😀
